### DISCUSSION TITLE
How to disable autocomplete with Enter key

### DISCUSSION THREAD

**Discussion by fvasquezl** [LazyVim/LazyVim#4093] [Q&A]
I have a python code, but I need remove enter autocomplete, just I need type enter and add new line

![image](https://github.com/user-attachments/assets/9ac1d7bc-47b2-46ab-bda6-ba41cb2a916c)

**Comment by dpetka2001** [comment]
Try this
```lua
{
  "hrsh7th/nvim-cmp",
  opts = function(_, opts)
    local cmp = require("cmp")
    
    opts.mapping = vim.tbl_deep_extend("force", opts.mapping, {
      ["<CR>"] = cmp.config.disable,
      ["<C-y"] = LazyVim.cmp.confirm({ select = true }),
    })
  end,
},
```
and change `<C-y>` to whatever keymap you would like to use for accepting the autocomplete.

**Reply by fvasquezl** [reply]
Sweet, but is possible to use [TAB] for moving inside box options for selection?

**Reply by dpetka2001** [reply]
I've not used Tab myself, so no idea. There are other discussions here or Recipes on `nvim-cmp` repo that have examples. Try to find them and see what works for you. I just showed what is the way to change the mappings.

**Reply by ssh352** [reply]
@dpetka2001 out of curiosity, do you use enter(the lazyvim default) instead of tab?

**Reply by dpetka2001** [reply]
Yes. Never used vscode or other editors so am not used to that behavior. I've only started learning programming earlier this year and Lazyvim was my first choice, so got used to that.
